Cargo.toml

```diff
@@ -43,6 +43,8 @@ sha2 = { version = "0.10.7", optional = true, features = ["oid"] }
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 js-sys = "0.3"
+# https://docs.rs/getrandom/0.2.16/getrandom/index.html#webassembly-support
+getrandom = { version = "0.2.16", features = ["js"] }
```

Comment:
> We only enable it if compiling for WASM, following getrandom's advice to not enable it unconditionally for our users.

Their docs (also) say this though:

> This feature should only be enabled for binary, test, or benchmark crates. Library crates should generally not enable this feature, leaving such a decision to users of their library.

I think this commit should be reverted, and CI adjusted to run WASM checks / tests with `--features getrandom/js`.